cli/cmd/lib_cluster_config.go

```diff
@@ -225,7 +225,7 @@ func getClusterUpdateConfig(cachedClusterConfig clusterconfig.Config, awsCreds A
 		userClusterConfig.Spot = cachedClusterConfig.Spot
 
 		if userClusterConfig.Spot != nil && *userClusterConfig.Spot {
-			err = userClusterConfig.AutoFillSpot(awsClient)
+			err = userClusterConfig.FillEmptyFields(awsClient)
```

Comment:
Since this only seems to fill spot fields, how do you feel about calling it `FillEmptySpotFields()`?

Reply:
Agreed. I wasn't happy with that single "fields" anyway.